app/views/products/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<body>
+<main>
+<section>
+  <h2><%= @product.name %></h2>
+    <p>Description: <%= @product.description.capitalize %></p>
+    <p>Price: $<%= @product.price %></p>
+    <p>Stock: <%= @product.stock %> units</p>
+    <p>Available: <%= @product.available %></p>
+    <p>User_id: <%= @product.user_id %></p>
+    <img src= <%= @product.photo_url %>>
```

Comment:
Once we have real photos, let's be sure to add an 'alt' tag. Maybe we can just make the 'alt tag' say the product name?

Reply:
Nice catch! I'm updating all the seeds with names, description, links to pictures, etc and I will update the view :)